src/MusicStore/Components/SystemClock.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace MusicStore.Components
+{
+    /// <summary>
+    /// Provides access to the normal system clock.
+    /// </summary>
+    public class SystemClock : ISystemClock
+    {
+        /// <inheritdoc />
+        public DateTime UtcNow
+        {
+            get
+            {
+                // The clock measures whole seconds only, to have integral expires_in results, and
```

Comment:
Looks fine but in the second sentence "the resolutions are" should be "resolution is" (no "the").